rmw_cyclonedds_cpp/src/serdata.cpp

```diff
@@ -514,6 +516,9 @@ static void sertype_rmw_free(struct ddsi_sertype * tpcmn)
     tp->type_support.type_support_ = NULL;
   }
 
+  free((void*)tp->type_information.data);
```

Comment:
These are allocated by `ddsrt_memdup`, right? Then they should be freed with `ddsrt_free` instead of free. (It won't matter on Linux, but on Windows it gets tricky with how they deal with DLLs.)

Reply:
My mistake. Change to `ddsrt_free`.